topcoffea/modules/corrections.py

```diff
@@ -89,17 +93,23 @@
 def AttachPerLeptonFR(leps, flavor, year=2018):
   if year == '2016APV': year = 2016
   for syst in ffSysts:
-    fr=SFevaluator['{flavor}FR_{year}{syst}'.format(flavor=flavor,year=year,syst=syst)](np.abs(leps.eta), leps.conept )
+    fr=SFevaluator['{flavor}FR_{year}{syst}'.format(flavor=flavor,year=year,syst=syst)](leps.conept, np.abs(leps.eta) )
     leps['fakefactor%s'%syst]=ak.fill_none(-fr/(1-fr),0) # this is the factor that actually enters the expressions
-  fr=SFevaluator['{flavor}FR_{year}'.format(flavor=flavor,year=year)](np.abs(leps.eta), leps.conept )
+  fr=SFevaluator['{flavor}FR_{year}'.format(flavor=flavor,year=year)](leps.conept, np.abs(leps.eta) )
```

Comment:
I have a quick question about this line. I'm not sure if I'm just missing something here, but it looks like this line is already covered by the for loop, since the loop includes `''` as one of the strings to loop over. If that's the case, I'm wondering if we could just remove the line?